dynawo/sources/Models/CPP/Controls/Voltage/ModelVoltageMeasurementsUtilities/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022-2022, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+
+##############################################
+#    DYNModelVoltageMeasurementsUtilities    #
+##############################################
+
+set(MODEL_NAME
+    DYNModelVoltageMeasurementsUtilities
+  )
+
+set(MODEL_SOURCES
+    DYNModelVoltageMeasurementsUtilities.cpp
+  )
+
+compile_cpp_model(${MODEL_NAME} "${MODEL_SOURCES}")
+
+target_include_directories(${MODEL_NAME}
+  INTERFACE
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+    $<TARGET_PROPERTY:dynawo_CPPModelsCommon,INTERFACE_INCLUDE_DIRECTORIES>
+  )
+target_include_directories(${MODEL_NAME} SYSTEM
+  PRIVATE
+  $<TARGET_PROPERTY:Boost::boost,INTERFACE_INCLUDE_DIRECTORIES>
```

Comment:
Maybe a stupid question, but why not the same indentation as above ?

Reply:
Jolly good question.